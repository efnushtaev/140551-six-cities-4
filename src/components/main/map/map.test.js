import React from 'react';
import MapContainer from './map';
import renderer from 'react-test-renderer';
import {expect, it} from '@jest/globals';


it(`Map should be rendered correct`, () => {
  const tree = renderer
    .create(<MapContainer/>,
        {
          createNodeMock: () => {
            return document.createElement(`div`);
          }
        })
    .toJSON();

  expect(tree).toMatchSnapshot();
});
