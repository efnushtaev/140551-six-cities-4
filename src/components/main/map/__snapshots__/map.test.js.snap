// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Map should be rendered correct 1`] = `
<div
  className="cities__map"
/>
`;
